File: Python Word examples/background/getBackgroundColor.py
import spirecloudword
from spirecloudword.configuration import Configuration as WordConfiguration

appId = "your id"
appKey = "your key"

configuration = WordConfiguration(appId, appKey)
api = spirecloudword.api.background_api.BackgroundApi(configuration)
color = api.get_backgroud_color("BackgroundSample.docx", folder="input")

File: JavaScript Word Examples/BackgroundApiDemo/getBackgroundColor.js
(function (){
    var appId = "your id";
    var appKey = "your key";
    var baseUrl = "https://api.e-iceblue.cn";

    var Spirecloudword = require('../../src/index');
    var apiClient = new Spirecloudword.ApiClient(appId, appKey,baseUrl);
    var instance = new Spirecloudword.BackgroundApi(apiClient);
    
    var name = "BackgroundDoc.docx";
    var opt = {folder: 'background'};
    var response = instance.getBackgroundColor(name, opt,
        function (error, data, response) {        
            if (error) throw error;
        });
})();